ax/plot/pareto_utils.py

```diff
@@ -555,7 +554,10 @@ def infer_reference_point_from_experiment(
     )
 
     # Transforming all the objectives to be maximized.
-    f_transformed = torch.tensor(multiplier, dtype=f.dtype, device=f.device) * f
+    f_transformed = (
+        torch.tensor(multiplier, dtype=f.dtype, device=f.device)
+        * f[:, obj_w.nonzero().view(-1)]
```

Comment:
Shouldn't the `view(-1)` should be a noop here?

Reply:
`obj_w.nonzero()` returns an `m x 1` tensor. Without the `view` the result ends up with an extra dimension.

Reply:
Looks like the docstring of `get_pareto_frontier_and_configs` is incorrect then? https://github.com/saitcakmak/Ax/blob/7e7b178e67aa7a1b061fc3dc34bf00000b078ed6/ax/modelbridge/modelbridge_utils.py#L798

Reply:
It's just the behavior of nonzero. `torch.rand(3).nonzero()` returns `tensor([[0], [1], [2]])`. If you use `as_tuple=True`, then it returns `(tensor([0, 1, 2]),)`. So, you either need to use the view to flatten the default or index into the tuple.

Reply:
Ah I see, thanks.